scripts/sct_compute_mtr.py

```diff
@@ -73,8 +75,16 @@ def main(args=None):
     convert(fname_mt0, os.path.join(path_tmp, "mt0.nii"), dtype=np.float32)
     convert(fname_mt1, os.path.join(path_tmp, "mt1.nii"), dtype=np.float32)
 
-    # go to tmp folder
+    # if changing output file name or location, create folder with mt0 and mt1 files at precised location
     curdir = os.getcwd()
+    if os.path.split(fname_mtr)[0] != curdir:
+        import shutil
```

Comment:
move imports to the top

Reply:
agreed, fixed in commit [16baeaf](https://github.com/YangHee-Min/spinalcordtoolbox/commit/16baeaf5c88a63412b5adca430da010a74fbd07a)